### ISSUE TITLE
[huggingface.co] Include a tag for mobile-compatible models.

### ISSUE THREAD

**Issue by dynamicwebpaige** [huggingface/huggingface_hub#741]
👋 Greetings! Am not sure if this is the correct place to file an issue for [huggingface.co](), but figured I'd try anyway. :)

### Issue Description
**Is your feature request related to a problem? Please describe.**
Would it be possible to include models that are mobile-compatible as a filterable category in the `Other` section on the Hugging Face website? Meaning `.tflite`, [`.ptl`](https://pytorch.org/mobile/home/) for PyTorch, or CoreML models for iOS devices.

<img width="385" alt="Screen Shot 2022-03-04 at 9 28 08 AM" src="https://user-images.githubusercontent.com/3712347/156812526-a81b55a1-3b22-4228-93cf-93183c4a578e.png">

## Additional context
In competitors' hubs, models of varying sizes are colocated on a single page ([example below](https://tfhub.dev/captain-pool/esrgan-tf2/1)) - so you might see the base example, a Colab notebook to test out the model interactively, and a TF Lite implementation. You can also sort and filter based on model type, as well as framework version and fine-tunability (though HuggingFace's models would win that contest every time 😃 ).

<img width="241" alt="Screen Shot 2022-03-04 at 9 36 55 AM" src="https://user-images.githubusercontent.com/3712347/156813780-21263ab8-79ce-47be-bcfc-49f6747dba22.png">

<img width="1104" alt="Screen Shot 2022-03-04 at 9 34 59 AM" src="https://user-images.githubusercontent.com/3712347/156813571-cdb6e2ab-3e35-4e0e-8262-54edef9fbbaa.png">

**Comment by osanseviero** [comment]
:thinking: we do have existing filters in "Libraries" when you click the +24, and TF Lite is actually one of the options over there https://huggingface.co/models?library=tflite&sort=downloads. Is that something similar to what you had in mind, but with additional file types?

**Comment by dynamicwebpaige** [comment]
@osanseviero : Yes, that would be perfect! Something that would let users know that if they need to deploy the model to an environment that isn't a laptop, or on a server, that they would be able to do so without finagling the quantization process themselves. 

And I love the work that @merveenoyan did to expand the filters for libraries! 😄

**Comment by osanseviero** [comment]
This is interesting. We've had discussions about increasing the number of models for mobile/edge devices. I think once we increase the number of models, adding a filter would definitively make sense. 

cc @julien-c @gary149 WDYT?

**Comment by gary149** [comment]
If we think it's important to easily filter/distinct mobile compatible models (no strong opinion on it), I understand @dynamicwebpaige point that it's more straightforward to have a "mobile ready" tag into the "others" section than having to expand/search in sub-sections. 

<img width="1042" alt="image" src="https://user-images.githubusercontent.com/3841370/157013053-350fbbb6-f6d6-47c3-ab82-930eebf1c87d.png">
